spec/factories/notifications.rb

```diff
@@ -1,30 +1,62 @@
+# frozen_string_literal: true
+
 FactoryGirl.define do
   factory :apns_notification, class: 'RubyPushNotifications::APNS::APNSNotification' do
-    tokens { [generate(:apns_token)] }
+    tokens {[generate(:apns_token)]}
```

Comment:
Why remove this whitespaces? They're actually required by hound

Reply:
This one is already resolved, i see that this one has an "outdated" flag. I've not used hound in accordance with github like this. but i've done changes related to hounds suggestions and in accordance with rubocop and rubymine. one of my submits had an aggressive change though, this was due to a plugin that changed some of my style settings without my knowledge, that was corrected. Is there any way to wipe the hounds history on this and re-evaluate the submitted codes current state?